tests/vasp/flows/test_mp.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+from pymatgen.core import Structure
+
+from atomate2.vasp.flows.mp import MPMetaGGARelax
+from atomate2.vasp.jobs.mp import (
+    MPMetaGGARelaxMaker,
+    MPPreRelaxMaker,
+)
+
```

Comment:
All inputs need to be within the tests themselves otherwise it breaks the tests. This is mentioned at the bottom of this page but perhaps we need to make this more clear elsewhere: https://materialsproject.github.io/atomate2/dev/vasp_tests.html

Reply:
I tried moving all imports except `pytest` into the test function yesterday which made no difference, tests were failing either way.